src/container_export.py

```diff
@@ -0,0 +1,522 @@
+                        process.stderr.strip(),
+                    )
+                    for container_id in container_ids:
+                        _unmount(
+                            os.path.join(all_containers_mount_root, container_id),
+                            log_file,
+                        )
+            except Exception as e:
+                logger.info(
+                    "Attempting to unmount each container under %s",
+                    all_containers_mount_root,
+                )
+                for container_id in container_ids:
+                    _unmount(
+                        os.path.join(all_containers_mount_root, container_id), log_file
+                    )
+
+        if os.path.exists(all_containers_mount_root):
+            try:
+                shutil.rmtree(all_containers_mount_root)
+            except OSError as e:
+                message = (
+                    "Could not remove temporary mount directories under "
+                    f"{all_containers_mount_root}: {e}"
+                )
+                logger.error(message)
+                log_entry(log_file, message)
+
+    logger.info("Returning %d output files", len(output_files))
+    return output_files
+
+
+@celery.task(bind=True, name=TASK_NAME, metadata=TASK_METADATA)
+def container_export(
+    self,
+    pipe_result: str = None,
+    input_files: list = None,
+    output_path: str = None,
+    workflow_id: str = None,
+    task_config: dict = None,
+) -> str:
+    """Export containers as raw disk images.
+
+    Args:
+        pipe_result: Base64-encoded result from the previous Celery task, if any.
+        input_files: List of input file dictionaries (unused if pipe_result exists).
+        output_path: Path to the output directory.
+        workflow_id: ID of the workflow.
+        task_config: User configuration for the task.
+
+    Returns:
+        Base64-encoded dictionary containing task results.
+    """
+    task_id = self.request.id
+    logger.info(
+        "Starting container export task ID: %s, Workflow ID: %s", task_id, workflow_id
+    )
+
+    final_output_files = []
+    temp_dir = None
+    disk_mount_dir = None
+    returned_disk_mount_dir = None
+
+    try:
+        input_files = get_input_files(pipe_result, input_files or [])
+
+        # Log file to capture logs.
+        log_file = create_output_file(
+            output_path,
+            extension="log",
+            display_name="container_export",
+        )
+        final_output_files.append(log_file.to_dict())
+
+        if not input_files:
+            message = "No input files provided."
+            logger.warning(message)
+            log_entry(log_file, message)
+
+            return create_task_result(
+                output_files=final_output_files,
+                workflow_id=workflow_id,
+            )
+
+        # We will only be processing one disk as container IDs are unique to each disk.
```

Comment:
This is fairly crucial, can you document that in the README? We can refactor this later to have the output file be a concatenation of e.g. input-file-name + container_ID to be able to process multiple files. 
Maybe even open an issue as a feature/enhacement for this so we do not forget.

Reply:
Updated README features and clearly stated that only the first disk is processed if multiple disks are provided.

Created an issue#7[0] to add support for multiple input disks.

[0] https://github.com/openrelik/openrelik-worker-containers/issues/7